docs/docs/cmd/spo/list/list-sensitivitylabel-ensure.mdx

```diff
@@ -41,19 +44,25 @@ This command is based on an API that is currently in preview and is subject to c
 
 ## Examples
 
-Applies a sensitivity label to a document library based on the list title.
+Applies a sensitivity label by id to a document library based on the list title.
+
+```sh
+m365 spo list sensitivitylabel ensure --webUrl 'https://contoso.sharepoint.com' --listTitle 'Shared Documents' --id 'Confidential'
```

Comment:
Hi @nanddeepn, the id is probably a GUID 😀

Reply:
Thanks for the correction 😀